consoles/sshXtermIPMI.pm

```diff
@@ -31,11 +32,40 @@ sub activate {
 
     my $testapi_console = $self->{testapi_console};
 
-    my @command = $self->backend->ipmi_cmdline;
-    push(@command, qw(sol activate));
-    my $serial = $self->{args}->{serial};
+    my $cstr              = "";
+    my @command_reset_tmp = $self->backend->ipmi_cmdline;
+    push(@command_reset_tmp, qw(mc reset cold));
+    my $command_reset = join(' ', @command_reset_tmp);
+
+    my @command_sleep_tmp = "sleep";
+    push(@command_sleep_tmp, qw(15));
```

Comment:
Why do we need a sleep of 15? I assume it is seconds, 15 seconds would be a lot of time wasted

Reply:
To make sure bmc already offline before starting ping.

Reply:
Yes, clearly. My question is: Can you ensure that with a proper check, not sleeping?